infra/spawn_pipeline_requirements.txt

```diff
@@ -1,4 +1,3 @@
-taskcluster==16.2.0
 json-e==3.0.0
-requests==2.22.0
-pyyaml==5.1.2
\ No newline at end of file
+pyyaml==5.1.2requests==2.22.0
```

Comment:
There is an error here

Reply:
fixed. Sorry, that got modified by the hook because I missed the file and did not realize it was affected